test/al_mock.js

```diff
@@ -94,6 +95,21 @@ const AZCOLLECT_CHECKIN_QUERY_COMPRESSED = {
     body : COMPRESSED_CHECKIN_BODY
 };
 
+const SEND_COLLECTOR_STATUS_BODY_DATA = {
+    status: "error",
+    inst_type: "collector",
+    stream: "Audit.Exchange",
```

Comment:
can we write testcases with multiple streams?

Reply:
we can send only one stream at a time. There is no restriction for stream it can be anything.